conf/logrotate.conf

```diff
@@ -115,15 +115,16 @@ notifempty
   daily
   rotate 5
   postrotate
-    st2ctl reopen-log-files st2garbagecollector
+    st2ctl reopen-log-files st2garbagecollector > /dev/null
   endscript
 }
 
 ## WebUI
 /var/log/st2/st2web.log {
```

Comment:
Right, it's outdated.
Please remove entire `st2web` block.

Current `st2web` logs are placed in `/var/log/nginx/` directory: https://github.com/StackStorm/st2/blob/master/conf/nginx/st2.conf#L21-L22 which is logrotated by `nginx`'s rules.